pbs_scripts/salmon_ti.pbs

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -l
+
+################################################################################
+#Script Name: salmon_ti.pbs
+#Desription : create transcriptome indices from cdna fasta
+#Args       : None 
+#Author     : Georgia Doing
+#Date       : 12-18-20
+#Email      : Georgia.Doing.GR@Dartmouth.edu
+################################################################################
+
+#PBS -q testq
+# #PBS -A NCCC
+#PBS -N sal_t_idx
+#PBS -l walltime=00:30:00
+#PBS -l nodes=1:ppn=1
+#PBS -m bea
+#PBS -M Georgia.Doing.GR@Dartmouth.edu
+#PBS -j oe
+#PBS -e oe
+#PBS -o oe
+
+
+
+cd $PBS_O_WORKDIR
+
+module load salmon
+
+salmon index -t ../pao1_cdna.fa.gz -i ./t_ind/pao1_cdna_k15 -k 15
```

Comment:
Just wanted to confirm that we're using the same indices to help track down what might be happening with the mapping rates. 

I downloaded the GENE DNA file for Pseudomonas aeruginosa PAO1 (Reference) and Pseudomonas aeruginosa UCBPP-PA14 from http://www.pseudomonas.com/strain/download

`Pseudomonas_aeruginosa_PAO1_107.ffn.gz`
`Pseudomonas_aeruginosa_UCBPP-PA14_109.ffn.gz`

Reply:
I downloaded the cDNA references from ensembleBacteria ([https://github.com/hoganlab-dartmouth/pa-seq-compendia/blob/main/t_indxs/ensemble_urls.txt](url))  . I think they  should be pretty similar to the gene-based ones from either ensemble or from your links above, but you are right that we should check. 

The links to ensemble that aren't ftp just for reference:
( [https://bacteria.ensembl.org/Pseudomonas_aeruginosa_pao1_gca_000006765/Info/Index](url)
[https://bacteria.ensembl.org/Pseudomonas_aeruginosa_ucbpp_pa14_gca_000014625/Info/Index](url) )